MvvmCross.Forms.Wpf/MvvmCross.Forms.Platforms.Wpf.csproj

```diff
@@ -21,7 +21,7 @@ This package contains the 'MvvmCross.Forms.Platforms.Wpf' libraries for MvvmCros
 
   <ItemGroup>
     <PackageReference Include="Xamarin.Forms" Version="3.0.0.482510" />
-    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="3.0.0.482510" />
+    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="3.0.0.530893" />
```

Comment:
Yeah, it was just left over from your Forms update PR.

Reply:
I just noticed that there's a ref in here for Xamarin.Forms as well as the Platform.WPF. Neither should be in here since we have a props file for Xamarin.Forms to ensure consistent versioning. I'll add this fix to a different PR so we don't forget it